### PR TITLE
Symfony 3 test suite

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,18 +14,21 @@
...
+        "symfony/finder": "^2.3|^3.0",
+        "symfony/filesystem": "^2.3|^3.0",
+        "symfony/yaml": "^2.3|^3.0",
+        "symfony/event-dispatcher": "^2.3|^3.0",
+        "symfony/property-access": "^2.3|^3.0"
     },
     "require-dev": {
-        "fabpot/php-cs-fixer": "1.4.2",
-        "phpunit/phpunit": "4.5.0"
+        "fabpot/php-cs-fixer": "dev-SF3",
```
Comment (mmoreram) [inline composer.json]: please, don't change the testing and formatting dependencies

File: composer.json [hunk 2]
```diff
@@ -14,18 +14,21 @@
...
+        "phpunit/phpunit": "^4.5"
     },
+    "repositories": [
+        { "type": "vcs", "url": "https://github.com/Ener-Getick/PHP-CS-Fixer" }
```
Comment (mmoreram) [inline composer.json]: Why?

File: .travis.yml [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
     - 5.6
     - 7.0
-    - hhvm-nightly
+    - hhvm
 
 matrix:
-    fast_finish: true
+    include:
+      - php: 5.6
+        env: SYMFONY_VERSION="2.8.*@dev symfony/phpunit-bridge:~2.7"
```
Comment (mmoreram) [inline .travis.yml]: Remove @ dev

File: .travis.yml [hunk 4]
```diff
@@ -1,17 +1,32 @@
...
+      - php: 5.6
```
Comment (mmoreram) [inline .travis.yml]: Check the indentation

File: .travis.yml [hunk 5]
```diff
@@ -1,17 +1,32 @@
...
+        env: SYMFONY_VERSION="3.0.*@dev"    
     allow_failures:
         - php: 7.0
-        - php: hhvm-nightly
+        - php: hhvm
+    fast_finish: true
+
+before_install:
+    - composer self-update
```
Comment (mmoreram) [inline .travis.yml]: not necessari

File: .travis.yml (hunk 3 above)
Comment (mmoreram) [inline .travis.yml]: Why phpunit-bridge?

**Comment by TomasVotruba** [comment]
Close due to no response timeout or too much work to finish.

**Pull request closed without merging**